python/lsst/qa/explorer/tractQADataIdContainer.py

```diff
@@ -0,0 +1,60 @@
+#
+# LSST Data Management System
+# Copyright 2008-2018 AURA/LSST.
+#
+# This product includes software developed by the
+# LSST Project (http://www.lsst.org/).
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.    See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the LSST License Statement and
+# the GNU General Public License along with this program.  If not,
+# see <http://www.lsstcorp.org/LegalNotices/>.
+#
+import argparse
+from collections import defaultdict
+
+from lsst.coadd.utils import CoaddDataIdContainer
+import lsst.pipe.base as pipeBase
+
+class TractQADataIdContainer(CoaddDataIdContainer):
+
+    def makeDataRefList(self, namespace):
+        """Make self.refList from self.idList
+        It's difficult to make a data reference that merely points to an entire
+        tract: there is no data product solely at the tract level.  Instead, we
```

Comment:
Haven't you just defined a data product solely at the tract level (`deepCoadd_qa_tract`)?

Reply:
good point! I'd just copied this docstring from one of the other DataRefContainers, and didn't change this.